### DISCUSSION TITLE
Why “latest” label on older version?

### DISCUSSION THREAD

**Discussion by nowsprinting** [openupm/openupm#1858] [Q&A] [answered]
This package's newest version is v2.1.3
But, “latest” label on v2.0.0

https://openupm.com/packages/com.vrmc.unigltf/?subPage=versions

**Comment by favoyang** [comment, marked as answer]
It's a bug when the monorepo repository doesn't have a correct setup with OpenUPM.

OpenUPM encourages the monorepo to use a separate Git tag for different packages, like `com.pkg.a/1.0.0` and `com.pkg.b/3.2.9`. To avoid the situation that one package bumps a version will trigger all monorepo packages to build (one of them will succeed and the rest failed for the version conflict reason).

For this package, https://openupm.com/packages/com.vrmc.unigltf/?subPage=pipelines
1. The pipelines try to build for Git tag v0.65.3, and expects version 0.65.3. But the actual version is [v2.1.3](https://github.com/vrm-c/UniVRM/blob/9e737d3d11c06bc3ed1f8d467b985ec17e5f9da2/Assets/UniGLTF/package.json).
1. It compares v0.65.3 with the latest version on the registry v2.0.0. 
1. Because v0.65.3 < v2.0.0, the pipelines think it's a patch, then tag it as `patch@0.65.3` (the way registry handles patch release).

Solution A: If the pipelines reject at step 1 for version mismatch (Git tag version not matched with the actual version of package.json), the package never gets build, because the repo doesn't have a matched Git tag for this package. That means monorepo has to create a separate Git tag for different packages. 

Solution B: If the pipelines ignore the expected version (from Git tag) but use the actual version to talk to the registry server, it will work but less strict.

Manually fixed `com.vrmc.unigltf` for now (2,1.3 as the latest). But it will break unless we implement one of the solutions above. I prefer solution A.

**Reply by nowsprinting** [reply]
Thanks!
I don't think it's an urgent issue.
Because normally, it will be imported with specifying the version from dependencies of UniVRM.

It's up to the maintainer of UniVRM to decide what to do in the future.
